blocks/footer/footer.css

```diff
@@ -1,18 +1,80 @@
 footer {
-  padding: 2rem;
-  background-color: var(--overlay-background-color);
+  padding: 2rem;   
+  background-color: #f0f0f0;
   font-size: var(--body-font-size-s);
 }
 
 footer .footer {
   max-width: 1200px;
-  margin: auto;
+  border:0; margin:0 auto; padding:0;
+}
+
+.footer > div > div {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  justify-content:center;
+}
+
+footer .footer ul {
+  list-style:none;
+  margin-left: auto;
+  display: flex;
+  overflow: hidden;
+  flex-flow: row wrap;
+  justify-content: flex-end;
+  flex-grow: 1;
+}
+
+footer .footer li {
+  border:0; margin:0; 
+  font-size: 14px;
+  padding: 0 2em;
+  text-align: center;
+  border-right: 1px solid black;
+}
+
+footer .footer li:last-child {
+  border: 0; 
+}
+
+footer .footer  a {
+  font-size: 14px;
+  text-decoration: none;
+  color: black;
+  text-transform: uppercase;    
+}
+   
+footer .footer a:hover {
+  color: #bd3826;
+}
+
+footer .footer a.active {
+  color: #bd3826;
+}
+
+footer .footer a:focus {
+  color: #bd3826;
+  outline: none;
 }
 
 footer .footer p {
   margin: 0;
+  padding-left: 10%;
```

Comment:
it seems padding is different on the original. any reason for using relative and the same all around?

![Screenshot 2023-01-16 at 15 27 12](https://user-images.githubusercontent.com/1872195/212701158-bc6302fd-4312-46d7-8ddc-482b54b898a7.png)
(left your's, right original)

Reply:
As the are not using flex to handle the spacing between the fist link and the second sort of navigation bar, this was the way I found to handle better the same behaviour that is shown on the original footer.

Reply:
please revisit, maybe in conjunction with this comment: https://github.com/hlxsites/elco-aerotop-sg/pull/26#discussion_r1071375293